kyo-core/jvm/src/main/scala/kyo/KyoAppPlatformSpecific.scala

```diff
@@ -0,0 +1,25 @@
+package kyo
+
+import scala.collection.mutable.ListBuffer
+
+abstract class KyoAppPlatformSpecific extends KyoApp.Base[Async & Resource & Abort[Throwable]]:
+
+    private val procListBuffer = new ListBuffer[() => Unit]
+
+    final override protected def run[A: Flat](v: => A < (Async & Resource & Abort[Throwable]))(using Frame): Unit =
+        import AllowUnsafe.embrace.danger
+        procListBuffer += (() =>
+            val resultEffect = handle(v).map { fiber =>
+                fiber.block(timeout).map(Abort.get(_))
+            }
+            val result = IO.Unsafe.run(Abort.run(resultEffect)).eval
+            if !result.exists(().equals(_)) then println(pprint.apply(result).plainText)
```

Comment:
Compiler will complain about strict equality of `A | Unit` if we use `!result.contains(_)` here, i guess that why it's been like this from the start.